packages/api-derive/src/staking/info.ts

```diff
@@ -34,13 +43,66 @@ function withStashController (api: ApiInterface$Rx, accountId: AccountId, contro
       stakingLedger: stakingLedger.isSome
         ? stakingLedger.unwrap()
         : undefined,
+      redeemable: redeemableSum(stakingLedger.unwrap().unlocking, eraLength, bestNumber),
```

Comment:
We are unwrapping it twice, or rather 2 times - so I propose moving stakingLedger out, i.e.

```js
const stakingLedger = stakingLedger.isSome ? stakingLedger.unwrap() | undefined
```

And passing that value around (not the null suggested earlier). Have a check, in the case of `isUndefined(stakingLedger)`, return `undefined`